LeetcodeProblems/Algorithms/Pascal'sTriangle.js

```diff
@@ -0,0 +1,71 @@
+// Leetcode 118
+
+// Problem: https://leetcode.com/problems/pascals-triangle/
+
+// Given an integer numRows, return the first numRows of Pascal's triangle.
+
+// In Pascal's triangle, each number is the sum of the two numbers directly above
+// Example 1:
+
+// Input: numRows = 5
+
+// Output: [[1],[1,1],[1,2,1],[1,3,3,1],[1,4,6,4,1]]
+/**
+
+ * @param {number} numRows
+
+ * @return {number[][]}
+
+ */
+
+var generate = function(numRows) {
+
+    var result = [];
+
+    
+
```

Comment:
Could you remove the return lines here and in the other places within the function? thanks!